tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -104,3 +105,32 @@ Is RHODS Version Greater Or Equal Than
     ${comparison} =  GTE  ${ver}  ${target}
     [Return]  ${comparison}
 
+Move To Installed Operator instance
+    [Arguments]   ${operator_name}     ${tab_name}      ${namespace}
+    Switch To Administrator Perspective
+    Navigate to Installed Operators
+    Installed Operators Should Be Open
+    Log To Console   ${namespace}
+    Run Keyword If  "${namespace}" == "None"   Select All Projects
```

Comment:
I think it is possible to use "Select Project By Name" (with the appropriate parameter set) instead of "Select All Projects".

Reply:
By default project is all project ..that's why i kept it if user want to select project we have that in else statement below

Reply:
I mean using `Select Project By Name  All Projects` instead of `Select All Projects`

Reply:
Done